### PR TITLE
Add disabled state to RichContenteditable and clean not-working v-model

### PR DESCRIPTION
**Pull request by skjnldsv**
In order: normal, `contenteditable=false`, `disabled=true`

image

### CODE REVIEW

**Comment by nickvergessen** [comment]
Conflict? Was it based on another PR?

**Comment by skjnldsv** [comment]
> Conflict? Was it based on another PR?

Oups, yes, it will change when the other pr is merged! :rocket:

**Review by ma12-co — APPROVED** [review]

File: src/components/RichContenteditable/RichContenteditable.vue [hunk 1]
```diff
@@ -31,7 +31,7 @@ This component displays contenteditable div with automated @ autocompletion [at]
 <template>
 	<div>
 		<RichContenteditable
-			v-model="message"
+			:value.sync="message"
```
Comment (skjnldsv) [inline src/components/RichContenteditable/RichContenteditable.vue:34]: Fixing the not-working v-model

File: src/components/RichContenteditable/RichContenteditable.vue [hunk 2]
```diff
@@ -352,7 +376,6 @@ export default {
 		updateValue(htmlOrText) {
 			const text = this.parseContent(htmlOrText)
 			this.localValue = text
-			this.$emit('input', text)
```
Comment (skjnldsv) [inline src/components/RichContenteditable/RichContenteditable.vue:355]: Fixing the not-working v-model

**Comment by skjnldsv** [comment]
Ping!

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**